tools/build/android/README.md

```diff
@@ -0,0 +1,36 @@
+# Build MixedReality-WebRTC for Android
+
+## Prerequisites
+
+1. Build the Chromium WebRTC base library. Instructions [here](../libwebrtc/README.md).
+
+2. Install prerequisites:
+    * Android Studio (https://developer.android.com/studio/index.html). Once installed, use the System Settings/Android SDK dialog to install the following packages:
+        * SDKs 9.0 (Pie), 8.1 (Oreo)
+        * CMake
+        * NDK 16.1.4479499
+    * JRE 1.8 (https://www.java.com/download/)
+
+3. Add environment variable JAVA_HOME pointing to your JRE install directory. e.g.: `JAVA_HOME="C:\Program Files\Android\Android Studio\jre"`
```

Comment:
On Linux that's `apt install openjdk-8-jre` and I think already installed by the Google dependency script. Shall we mention it for completeness?

Reply:
I noted later that this is also needed on Unity, although Unity will strongly suggest using the version shipped with the Unity Editor. Maybe we should recommend that too, or at least point it out.